ui/static/ui/js/listing_resources.jsx

```diff
@@ -430,6 +432,7 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
     var ListingPage = React.createClass({
       render: function() {
         return <div>
```

Comment:
We should avoid `id` whenever possible since they may conflict and we would prevent easy reuse of components. CSS classes are fine but I prefer to add those when we have something to style, or maybe as identification for tests.